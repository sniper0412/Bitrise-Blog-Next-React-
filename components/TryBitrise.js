import React from 'react';

const TryBitrise = () => (
  <div className="try-wrapper">
    <div className="container">
      <h2>Try Bitrise</h2>
      <h1>Automate your building, testing and deployment</h1>
      <a
        href="https://www.bitrise.io/users/sign_up?utm_source=bitriseblog&utm_medium=web&utm_campaign=static"
        className="start-building-button"
      >
        Start building for free
      </a>
    </div>
  </div>
);

TryBitrise.propTypes = {};

export default TryBitrise;
