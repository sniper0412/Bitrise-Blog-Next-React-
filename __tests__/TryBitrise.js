import renderer from 'react-test-renderer';

import TryBitrise from '../components/TryBitrise';

describe('TryBitrise component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<TryBitrise />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
