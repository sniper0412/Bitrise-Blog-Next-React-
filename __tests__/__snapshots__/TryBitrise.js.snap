// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TryBitrise component renders correctly 1`] = `
<div
  className="try-wrapper"
>
  <div
    className="container"
  >
    <h2>
      Try Bitrise
    </h2>
    <h1>
      Automate your building, testing and deployment
    </h1>
    <a
      className="start-building-button"
      href="https://www.bitrise.io/users/sign_up?utm_source=bitriseblog&utm_medium=web&utm_campaign=static"
    >
      Start building for free
    </a>
  </div>
</div>
`;
